runtime/src/main/java/io/quarkus/reactivemessaging/http/runtime/QuarkusHttpConnector.java

```diff
@@ -101,8 +105,19 @@ private HttpMethod getMethod(String methodAsString) {
         }
     }
 
+    private HttpVersion getProtocolVersion(String versionAsString) {
+        try {
+            return HttpVersion.valueOf(versionAsString);
+        } catch (IllegalArgumentException e) {
+            String error = "Unsupported HTTP protocol version: " + versionAsString + ". The supported versions are: "
+                    + String.join(", ", Arrays.stream(HttpVersion.values()).map(HttpVersion::name).toList());
```

Comment:
Why not just Arrays.toString(HttpVersion.values())?

Reply:
Thanks, improved.